app/src/main/scala/Main.scala

```diff
@@ -32,8 +31,8 @@ object Main {
       p("Stuff about registering too")
     )
 
-    val backend         = FetchBackend()
-    val healthEndpoints = new HealthEndpoints {}
+    val runtime = zio.Runtime.default
+    val client  = BackEndClientLive.live
 
     // When we click a button, we will emit a Future that will fetch the current time
     val timeBus: EventBus[Future[Instant]] = new EventBus
```

Comment:
Is there a way to convert a ZIO stream to this event bus? I think we may overcome the Future limitation which I agree is a bit clunky